.github/workflows/test_on_deploy.yml

```diff
@@ -0,0 +1,32 @@
+name: Run Integration Tests on Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version: [3.8, 3.9, 3.10, 3.11]
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+          pip install -r requirements_test.txt
+
+      - name: Run integration tests
+        run: |
```

Comment:
let's add this env variable 
https://github.com/mindsdb/minds_python_sdk/blob/main/tests/integration/test_base_flow.py#L15